conda-recipe/meta.yaml

```diff
@@ -21,7 +21,6 @@ requirements:
     - {{ compiler('cxx') }}
   host:
     - python
-    - pip
```

Comment:
This should remain in `host` section. It is a build dependency used in `script` above.

Reply:
Thank you for noticing! Fixed.